ZEP/instructions/zep0000.md

```diff
@@ -0,0 +1,201 @@
+
+## ZEP Workflow
+
+The ZEP process begins with a new idea for Zarr. It is highly recommended that a single ZEP contain a single key proposal or new idea. Small enhancements or 
+patches often don’t need a ZEP and can be injected into the Zarr development workflow with a pull request to the Zarr ZEP repo. The more focused the ZEP, 
+the more successful it tends to be. If in doubt, split your ZEP into several well-focused ones.
+
+Each ZEP must have a champion—someone who writes the ZEP using the style and format described below, shepherds the discussions in the appropriate forums, 
+and attempts to build community consensus around the idea. The ZEP champion (a.k.a. Author) should first attempt to ascertain whether the idea is suitable for a ZEP. 
+Posting to the [Gitter](https://gitter.im/zarr-developers/community) is the best way to go about doing this.
+
+Vetting an idea publicly before going as far as writing a ZEP is meant to save the potential author time. Asking the Zarr community first if an idea is original 
+helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussion. It also helps to make sure the idea applies to 
+the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people.
+
+Once the champion has asked the Zarr community whether an idea has any chance of acceptance, a draft ZEP should be presented to the appropriate venue mentioned 
+above. This gives the author a chance to flesh out the draft ZEP to make properly formatted, of high quality, and to address initial concerns about the proposal.
+
+After the PR for the ZEP is in place, a post should be made on the Gitter channel containing the sections up to “Backward compatibility”, to limit discussion 
+there to usage and impact. Discussion on the pull request will have a broader scope, also including details of implementation.
+
+Standards Track ZEPs consist of two parts, a design document and a reference implementation. It is generally recommended that at least a prototype implementation 
+be co-developed with the ZEP, as ideas that sound good in principle sometimes turn out to be impractical when subjected to the test of implementation. 
+Often it makes sense for the prototype implementation to be made available as PR to the Zarr repo (making sure to appropriately mark the PR as a WIP).
+
+### Submitting a ZEP
+
+The proposal should be submitted as a draft ZEP via a GitHub pull request to the zarr-developers/zep directory with the name `zep-<n>.rst` where `<n>` is an appropriately assigned four-digit number (e.g., zep-0000.rst). The draft ZEP must use the [ZEP X - Template and Instructions](https://github.com/MSanKeys963/governance/blob/master/ZEP/template/template.md) file.
```

Comment:
```suggestion
The proposal should be submitted as two pull requests to the `zarr-developers/zarr-specs` repo.
The first PR should contain narrative text of the ZEP in the `zep` directory with the name `zep-<n>.rst` where `<n>` is an appropriately assigned four-digit number (e.g., zep-0000.rst). The draft ZEP must use the [ZEP X - Template and Instructions](https://github.com/MSanKeys963/governance/blob/master/ZEP/template/template.md) file.
For Specification ZEPs, there should be a second PR with the actual proposed changes to the spec.
```

Reply:
I assume if it's happening to a single repo, it could be one PR, no?

Reply:
That depends on whether we care about "publishing" the ZEP to a website before it is accepted (as done with [NEPs](https://numpy.org/neps/). In that case, we would merge the ZEP text PR first as a "Status: Draft" NEP, while the actual PR implementing the change would remain umerged until the ZEP is accepted.

Alternatively, we may decide that the process of publishing the ZEPs as standalone documents is not necessary for Zarr, and instead the PR, including its description, effectively IS the ZEP. I would not be opposed to that. It might be easier to manage.

Reply:
Resolved this at the ZSC meeting on 14/4. The decision is to have a separate repo `zep` which would render and display the ZEPs on the webpage which means draft ZEPs would also be displayed on the website with the status as `Draft`.